### DISCUSSION TITLE
How add my user authorize feature in Elsa Studio?

### DISCUSSION THREAD

**Discussion by anhlee97** [elsa-workflows/elsa-studio#253] [Q&A]
I want the functional rights for my users in Elsa Studio. For example, User can only view, User B will be able to edit and delete.

**Comment by glime-ai** [comment]
To implement functional rights for users in Elsa Studio, such as view-only or edit and delete permissions, you would typically need to manage user roles and permissions within the application. However, based on the context provided, there is no direct mention of built-in user role management or access control features specifically for Elsa Studio in the provided sources.

Elsa Workflows is an extensible framework, and you can potentially implement custom user roles and permissions by extending the application. This could involve creating custom activities or integrating with an existing identity management system that handles user roles and permissions.

Since the documentation and sources provided do not explicitly cover this feature, you may need to look into the following options:

1. Check the official Elsa Workflows documentation for any updates or plugins related to user management and access control.
2. Explore the possibility of contributing to the project by implementing this feature and submitting a pull request, as suggested by the roadmap and contribution guidelines on GitHub.
3. Engage with the Elsa community on forums, GitHub issues, or Stack Overflow to see if others have implemented similar features or to request guidance on how to achieve this.

If you require further assistance or specific guidance on implementing such a feature, it would be best to reach out to the Elsa Workflows community or maintainers directly for more detailed support.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Comment by anhlee97** [comment]
We have an IAM login system that can share permissions there. I have 2 questions 
1. Is it possible to integrate login with IAM on Elsa Studio?
2. How to add functional rights to Elsa Studio?
